src/Mariuzzo/LaravelJsLocalization/Commands/LangJsCommand.php

```diff
@@ -85,6 +85,6 @@ protected function getOptions()
      */
     public function getPublicPath()
     {
-        return $this->laravel['public_path'];
+        return public_path();
```

Comment:
All good! Tests are broken but the package is working. I should keep investigating how to fix those tests...